docs/developer-guide.md

```diff
@@ -68,7 +68,7 @@ The docker image built from previous step has embedded some important build info
 these info with docker `inspect` command.
 
 ```
-docker inspect yunikorn/yunikorn-scheduler-k8s:latest
+docker inspect yunikorn/yunikorn:scheduler-latest
```

Comment:
shouldn't this be `apache/yunikorn:scheduler-latest`?

Reply:
you are right! Just changed it